### PR TITLE
chore: set up next frontend

### PR DESCRIPTION
**Pull request by malwaregarry**
Used create next-app to set up frontend

### CODE REVIEW

File: frontend/.gitignore
Comment (francisyzy) [inline frontend/.gitignore:1]: Do we need individual .gitignore in the sub dir or just the one main one at root?

File: frontend/.prettierrc.json
Comment (francisyzy) [inline frontend/.prettierrc.json:1]: should we stick w this `preetierrc` for the rest of the TS/JS projects?

Comment (malwaregarry) [reply frontend/.prettierrc.json:1]: this is just a simple one i found, do you have another one that you prefer? we can change to that

but i do think we should standardise our formatting.

File: frontend/.gitignore
Comment (malwaregarry) [reply frontend/.gitignore:1]: i think individual ones are good as we are using this as a monorepo. what do you think?

File: frontend/.prettierrc.json
Comment (francisyzy) [reply frontend/.prettierrc.json:1]: I don’t have any I prefer, its fine to stick w this. How should we proceed?

File: frontend/.gitignore
Comment (francisyzy) [reply frontend/.gitignore:1]: Sure, can you help remove the .gitignore @ root?

File: frontend/.prettierrc.json
Comment (malwaregarry) [reply frontend/.prettierrc.json:1]: ok let's keep this for now and change if we find something wrong with it

File: frontend/.gitignore
Comment (malwaregarry) [reply frontend/.gitignore:1]: instead of removing it entirely, i updated the root gitignore to keep editor configs (vscode). what do you think?

**Review by francisyzy — APPROVED** [review]

**Pull request merged**